### PR TITLE
Sl6.x: add new perl dependencies for Quattor development

### PR DESCRIPTION
**Pull request by jouvin**
`perl-Module-Load`, `perl-Parallel-ForkManager`, `perl-Text-Diff` and `perl-Perl-Critic` added (required by some unit tests).

### CODE REVIEW

**Comment by jrha** [comment]
What is this exactly?

File: rpms/quattor-development.pan [hunk 1]
```diff
@@ -22,6 +22,8 @@ unique template rpms/quattor-development;
   pkg_repl('perl-Taint-Runtime');
   pkg_repl('perl-enum');
   pkg_repl('perl-XML-Simple');
+  pkg_repl('perl-Module-Load');
+  pkg_repl('perl-Perl-Critic');
```
Comment (stdweird) [inline rpms/quattor-development.pan]: it needs a version that is more recent than epel has. i'm going to add the correct version to the externals repo asap.

Comment (jrha) [inline rpms/quattor-development.pan]: New version is in externals: `'perl-Perl-Critic', '1.126-4.rhel5', 'noarch'`

**Comment by jouvin** [comment]
I got the new version as soon as the the `quattor_externals` has been updated. The template is ok (I don't think this is a good idea to specify a version in it, as is should be a minimum rather than a strict version).

**Pull request merged**